usaspending_api/conftest_helpers.py

```diff
@@ -103,16 +104,28 @@ def __init__(self, index_type):
             "write_alias": self.index_name + "-load-alias",
             "process_deletes": True,
         }
+
+        match self.index_type:
+            case "award":
+                es_id_field = "award_id"
```

Comment:
Do these need to be different variables since they are always the same?

Reply:
I consolidated these variables. I originally left them separate in the future case that we wanted to use a non-primary key field as the ES `_id` field's value it would be more obvious.